packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -73,6 +73,10 @@ export type TileProps = {
    * Dropdown menu containing `MenuItem`s.
    */
   moreOptions?: React.ReactNode[];
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
```

Comment:
Do we need to add status and loading icons as in css repo?
![image](https://user-images.githubusercontent.com/83585998/197762890-9bdd3664-27f2-4593-b3c7-1dbc7cc0dd95.png)

Reply:
Yes, I feel so. It gives a 1-1 mapping between this and the CSS repo. Also, it looks better. Also, other aspects of the tile seem to be almost identical to the css examples, so would be better that this is also based off of the css examples.

Reply:
Added icons.